libraries/adaptive-expressions/package.json

```diff
@@ -28,7 +28,8 @@
     "lodash": "^4.17.15",
     "lru-cache": "^5.1.1",
     "moment": "2.24.0",
-    "moment-timezone": "^0.5.25"
+    "moment-timezone": "^0.5.25",
+    "nyc": "^15.0.0"
```

Comment:
Should this be a devDependency?

Reply:
Absolutely. Should've caught that initially. Fixed.